java/1.2/modules/ROOT/pages/configure-java-module-acb-edit.adoc

```diff
@@ -0,0 +1,331 @@
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-configure-attributes"]
+
+[[configure-trigger]]
+== Configure a Trigger for the Flow
+A trigger (source) starts the flow when specified conditions are met. If a connector has its own associated sources, you can use a connector-specific trigger to initiate the flow. Each connector trigger has a unique configuration.
+
+You can configure the following sources for this connector:
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+[[configuring-new]]
+=== Configuring the New Operation
+
+Create a new class instance by using the *New* operation of the Java module. The identifier of a constructor includes the class and constructor names, and the constructor has a full description of its signature, including the types of each parameter.
+
+For example, if you invoke the constructor `Me(String name, int age)`, which belongs to the `org.bar.Me` class, then the identifier of the method is `Me(String,int)`.
+
+To configure the attributes for *New* in the Anypoint Code Builder UI:
+
+. Click the *New* component on the canvas.
+. In the *General* tab:
+.. In *Class*, search and select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Constructor*, search and select the constructor to use. 
++
+NOTE: If this is blank, click the refresh icon to populate it.
+. Select the *Advanced* tab to configure the following attributes: 
++
+[%header%autowidth.spread]
+|===
+| UI Field | Description | Required
+| Target variable | Name of the variable to assign the class instance to. | No
+| Target value | Expression that evaluates the operation's output. The expression outcome is stored in the target variable. | No
+| Error Mappings a| Creates custom error types that you map to Mule error types. The mappings can help you further differentiate and group errors in your app. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:mule-runtime::mule-error-concept.adoc[]. | No
+|===
+. Select the *Input/Output* tab to filter and view input and output variables stored in DataWeave.
+
+After you add your class instance, <<configuring-invoke,configure the *Invoke* operation>>.
+
+[[configuring-invoke]]
+=== Configuring the Invoke Operation
+
+The *Invoke* operation invokes methods with the provided arguments using a specified class instance. The identifier of the method to invoke includes the class and method names.
+
+To configure the attributes for *Invoke* in the Anypoint Code Builder UI:
+
+. Click the *Invoke* component on the canvas.
+. In the *General* tab:
+.. Select the class *Instance* for invoking the method.
++
+This is the target variable assigned to the *New* operation.
+.. In *Class*, select the Java class that contains the method to use. 
```

Comment:
Different from line 49?
If not, create and use partials for these duplicated steps?